qa/rpc-tests/feature_fedpeg.py

```diff
@@ -47,27 +49,61 @@ def __init__(self):
         self.setup_clean_chain = True
         self.num_nodes = 4
 
+    def add_options(self, parser):
+        parser.add_option("--parent_binpath", dest="parent_binpath", default="",
+                          help="Use a different binary for launching nodes")
+        parser.add_option("--parent_bitcoin", dest="parent_bitcoin", default=False, action="store_true",
+                          help="Parent nodes are Bitcoin")
+
+    def start_node(self, idx):
```

Comment:
I'm not so sure this adds a lot of readibility, you could just use:

```
start_node(n, self.options.tmpdir, self.extra_args[n], chain='sidechain')
```
in one place and

```
binary = self.options.parent_binpath if self.options.parent_binpath != "" else None
chain = "regtest" if self.options.parent_bitcoin else "parent"
start_node(n, self.options.tmpdir, self.extra_args[n], binary=binary, chain=chain)
```
In the other, at least for the sidechains case seems unnecessarily complicated.

Reply:
I also use it to restart the node lateron.